lib/galaxy/webapps/galaxy/api/tool_data.py

```diff
@@ -24,12 +146,25 @@ def index(self, trans, **kwds):
 
         Return a list tool_data tables.
         """
-        return list(a.to_dict() for a in self._data_tables.values())
+        return self.tool_data_manager.index()
 
     @web.require_admin
     @expose_api
     def show(self, trans, id, **kwds):
-        return self._data_table(id).to_dict(view='element')
+        """
+        GET /api/tool_data/{id}
+
+        Get details of a given tool data table.
+
+        :type   id:     str
+        :param  id:     the id of the data table
+        """
```

Comment:
I wouldn't worry about adding docs to API endpoints we're immediately deprecating in the same PR 😁.

Reply:
You are right 😅